src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-view-sidebar/overview/Overview.scss

```diff
@@ -0,0 +1,171 @@
+@import 'src/styles/common';
+
+.titleWithSeparator {
+  color: $dark-grey;
+  font-size: 12px;
+  border-bottom: 1px solid $grey;
+  margin-bottom: 15px;
+  position: relative;
+  margin-top: 20px;
+}
+
+.geneDetails {
+  display: grid;
```

Comment:
Why is it a grid rather than just a div with two spans?
I guess, the test of which layout to choose is the question of how this line will behave when it has to wrap to the next line. If it will wrap thus:

```
very long gene name   very long
                      transcript name
```

then it's a grid; if it will behave thus:

```
very long gene name   very long
  transcript name
```
then it's just a sequence of spans.

I suspect that, given that gene name can be arbitrarily long, the grid is not the best option for this layout.

Reply:
By looking at the spec I guess it should behave like it is in the second example. I have updated it to use spans.